subliminal/providers/legendastv.py

```diff
@@ -229,7 +229,8 @@ def search_titles(self, title):
                     title['season'] = int(source['temporada'])
                 else:
                     match = season_re.search(source['dsc_nome_br'])
-                    title['season'] = int(match.group('season'))
+                    if match:
+                        title['season'] = int(match.group('season'))
```

Comment:
Please add a warning if the match failed as I believe this is quite unexpected, right?